src/components/pages/Triggers/Triggers.tsx

```diff
@@ -83,16 +62,17 @@ const Triggers: React.FC = () => {
 
   const [updateTriggers] = useUpdateTriggersMutation();
 
-  const [form] = Form.useForm();
+  const [form] = Form.useForm<TriggersFormValues>();
 
-  const [defaultFormattedTriggers, setDefaultFormattedTriggers] = useState<any>([]);
+  const [defaultFormattedTriggers, setDefaultFormattedTriggers] = useState<TestTriggerFormEntity[]>([]);
 
   const appNamespace = useAppSelector(selectNamespace);
   const executors = useAppSelector(selectExecutors);
 
-  const setDefaultTriggersData = (_triggersList?: TestTrigger[]) => {
-    if (_triggersList) {
-      const triggersData = _triggersList.map(trigger => {
+  const setDefaultTriggersData = (_triggersList: TestTrigger[]) => {
+    if (_triggersList && _triggersList.length) {
+      // @ts-ignore
+      const triggersData: TestTriggerFormEntity[] = _triggersList.map(trigger => {
```

Comment:
`@ts-ignore` should not be there. Should this function have the option to return `null`, or something is wrong with it?

```suggestion
      const triggersData: (TestTriggerFormEntity | null)[] = _triggersList.map(trigger => {
```